extensions/wikia/CreateNewPageFlow/scripts/flowTracking.js

```diff
@@ -0,0 +1,34 @@
+define('flowTracking', ['wikia.cookies', 'wikia.tracker', 'mw'], function (cookies, tracker, mw) {
+	var cookieName = 'CNPFlowLabel',
+		track = tracker.buildTrackingFunction({
+			action: tracker.ACTIONS.CLICK,
+			trackingMethod: 'analytics',
+			category: 'create-new-page-flows'
+		});
+
+	function beginFlow(label) {
```

Comment:
Please add docblocks to public methods - if label stands for name of the flow, it can also be used to differentiate between cookies related to different flows